File: src/main/java/com/coffee/shop/api/demo/image/api/model/Image.java
package com.coffee.shop.api.demo.image.api.model;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Lob;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.hibernate.annotations.Type;

@Entity
@Data
@NoArgsConstructor
public class Image {

    @Id
    private Long id;
    private String type;
    @Lob
    @Column(columnDefinition = "BLOB")
    @Type(type = "org.hibernate.type.BinaryType")
    private byte[] content;
    private String destination;
}

File: src/main/kotlin/com/coffee/shop/api/demo/image/api/model/Image.kt
package com.coffee.shop.api.demo.image.api.model

import javax.persistence.Entity
import javax.persistence.Id

@Entity
class Image {
    @Id
    var id: Long? = null
    var type: String? = null
    var name: String? = null
    var destination: String? = null
}